Cython/Compiler/Options.py

```diff
@@ -326,6 +326,10 @@ def normalise_encoding_name(option_name, encoding):
     'c_string_type': one_of('bytes', 'bytearray', 'str', 'unicode'),
     'c_string_encoding': normalise_encoding_name,
     'trashcan': bool,
+    'dataclasses.dataclass': Ellipsis,  # use Ellipsis as a flag not to attempt to analyse the arguments
+                            # when the directive is applied
```

Comment:
why not?

Reply:
Comment is updated slightly to clarify the reasoning.